sihl/src/sihl.mli

```diff
@@ -956,7 +956,7 @@ module Database : sig
       [None] if an unsupported database is used. *)
   val used_database : unit -> database option
 
-  val start : unit -> unit Lwt.t
+  val start : ?skip_default_pool_creation:bool -> unit -> unit Lwt.t
```

Comment:
Can you add some documentation to this? (when to use it)

Reply:
Fixed.